bitnami/redis/values-production.yaml

```diff
@@ -712,3 +712,11 @@ podSecurityPolicy:
   ## Specifies whether a PodSecurityPolicy should be created
   ##
   create: false
+
+## Define a disruption budget
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+##
+podDisruptionBudget:
+  enabled: false
+  # minAvailable: 1
```

Comment:
Would it make sense to uncomment `minAvailable` so it defaults to 1 when the `podDisruptionBudget.enabled` flag is set?

Reply:
Yep, i think it's reasonable. 

But maybe setting `maxUnavailable: 1` would be a better option? What do you think?